runtime/common/src/migrations.rs

```diff
@@ -364,11 +273,11 @@ where
 		//	RemoveMinBondForOrbiterCollators::<Runtime>(Default::default());
 
 		// RT2700
-		let missing_balances_migrations = MissingBalancesMigrations::<Runtime>(Default::default());
-		let fix_pallet_versions =
-			FixIncorrectPalletVersions::<Runtime, Treasury, OpenTech>(Default::default());
-		let pallet_referenda_migrate_v0_to_v1 =
-			PalletReferendaMigrateV0ToV1::<Runtime>(Default::default());
+		// let missing_balances_migrations = MissingBalancesMigrations::<Runtime>(Default::default());
+		// let fix_pallet_versions =
+		// 	FixIncorrectPalletVersions::<Runtime, Treasury, OpenTech>(Default::default());
+		// let pallet_referenda_migrate_v0_to_v1 =
+		// 	PalletReferendaMigrateV0ToV1::<Runtime>(Default::default());
```

Comment:
We could probably remove these lines to keep this function small, just keeping the ones bellow should be enough to go back a look at what those migrations did. 

Nothing important.